source/_docs/pantheon-yml.md

```diff
@@ -36,6 +36,36 @@ protected_web_paths:
 * You may not be able to protect files or paths with special characters
 * Wait a few seconds for changes to take effect
 
+### Enforce HTTPS + HSTS
+
+HTTPS adds a layer of encryption that prevents others from snooping on or tampering with traffic to your site. HTTP Strict Transport Security (**HSTS**) instructs browsers to only connect via HTTPS and helps protect websites against protocol downgrade attacks and cookie hijacking.
+
+<div class="alert alert-info" role="alert">
+<h4 class="info">Note</h4>
+<p markdown="1">Before adjusting `enforce_https`, review and understand the configuration options and all considerations to avoid unintended consequences.</p>
+</div>
+
+Ensure that your site will always use HTTPS to deliver content with `enforce_https`. Five values are available, from least to most secure:
```

Comment:
Suggest adding the following block: 

```
enforce_https: full+subdomains
``` 

^ demonstrate valid syntax as a key-value pair (vs an array) and align new section w/existing (all other configs have a code example nearby)